apps/mochi-web/components/Profile/PaylinkSection.tsx

```diff
@@ -0,0 +1,71 @@
+import { Badge, Button, ColumnProps, Table, Typography } from '@mochi-ui/core'
+import { format } from 'date-fns'
+import { useFetchPayRequests } from '~hooks/profile/useFetchPayRequests'
+import { useProfileStore } from '~store'
+import { ModelPayRequest } from '~types/mochi-pay-schema'
+import { Amount, PaylinkUrl } from './TableColumns'
+
+const Action: ColumnProps<ModelPayRequest>['cell'] = (props) =>
+  props.row.original.claim_tx ? (
+    <Badge appearance="black" label="Claimed" />
+  ) : (
+    <Button color="success" size="sm">
+      Claim
+    </Button>
+  )
+
+export const PaylinkSection = () => {
+  const { me } = useProfileStore()
+  const { data: requests = [], isLoading } = useFetchPayRequests({
+    profile_id: me?.id,
+    entity: 'sender',
+    type: 'paylink',
+  })
+
+  return (
+    <div className="space-y-2">
+      <div className="py-2">
+        <Typography level="h7">Pay Link</Typography>
+      </div>
+      <Table
+        border
+        isLoading={isLoading || !me?.id}
+        data={requests}
+        columns={[
```

Comment:
Can you specify the width for the column to resemble the design plus add a min-width for the container to support horizontal scrolling on mobile?